.github/workflows/typescript-pr-check.yml

```diff
@@ -0,0 +1,28 @@
+#
+# Copyright (c) 2020 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+
+name: typescript
+
+on: [push, pull_request]
```

Comment:
Do we need to run this for all changes? The majority of PR's in the plugin registry relate to sidecars or plugin updates, so running this check on every PR might be unnecessary. How about running it only on `**.ts` files?

Reply:
yes but 'ot only on .ts but as well if you modify package.json as well

Reply:
So `**.ts` and `**/package.json` then :+1: